libvast/src/detail/env.cpp

```diff
@@ -0,0 +1,36 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2021 The VAST Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include "vast/detail/env.hpp"
+
+#include <cstdlib>
+#include <mutex>
+
+namespace vast::detail {
+
+namespace {
+
+auto env_mutex = std::mutex{};
```

Comment:
What do you think in general about addind a `s_` prefix for the name of static data? For these small functions its obvious, but in general i think its nice when access to global data really stands out while reading the source code. In particular once classes get thrown into the mix, where a non-locally declared variable might be either a data member or a global variable.

Reply:
I will add a comment above the variable that makes it clear that this is global to this translation unit.